source/guide/extend-module-using-swift.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Extend weex using swift
```

Comment:
Using "Extend iOS with swift" looks good and can keep consistent with other titles.